components/hyper_serde/tests/tokens.rs

```diff
@@ -32,7 +34,6 @@ fn test_cookie() {
     // string with a bunch of indices in it which apparently is different from the exact same
     // cookie but parsed as a bunch of strings.
     let cookie: Cookie = Cookie::build("Hello", "World!")
-        .max_age(Duration::seconds(42))
```

Comment:
@mrobinson Cookie crate uses `time` crate and  max_age takes `time:: Duration`, I removed it and adjusted the test, what do you think?

Reply:
Is this to avoid the conflict in versions? I think it's okay to depend on the new version of `time` (0.3, I think) in order to use cookie in this way. We are already pulling it in if it's a dependency.

Reply:
If you need to import more than one version of `Duration` into the same code units you can do something like:

```rust
use std::time::Duration as StdDuration;
```